src/models/lineChart.js

```diff
@@ -379,7 +379,7 @@ nv.models.lineChart = function() {
                         allData[indexToHighlight].highlight = true;
                 }
 
-                var xValue = xAxis.tickFormat()(chart.x()(singlePoint,pointIndex));
+                var xValue = tooltip.headerFormatter()( chart.x()(singlePoint,pointIndex) );
```

Comment:
It seems to me that it shoudl just be sending the unformatted value since the tooltip calls headerFormatter already... the bug is really that we were doing tickFormat() at all there...